### ISSUE TITLE
error on flash

### ISSUE THREAD

**Issue by gurkburk76** [schmurtzm/RFLink32-For-Sonoff-RF-Bridge#9]
So not sure if i'm being stopid here or what's going on.
Afaik if you never used rflink before you need to flash esp32-full.img from https://github.com/cpainchaud/RFLink32/releases

I'm flashing with esptool 4.2.1 on windows since the python was just doing dumb shit (tm)

I used esptool.exe --chip auto --baud 460800 write_flash -z 0x1000 esp32-full.img
I also tried to change 0x1000 to 0x10000 as i saw that it was needed for some other flashing tool, not sure if it matters, i'm guessing it's something to tell where on the flash to start writing.
In the flashing guide text file it does say 0x10000 but that's with the OTA image so it might differ.. what do i know :)
"Using Esptool
python.exe esptool.py --chip esp32 --baud 460800 write_flash -z 0x10000 esp32-firmware-OTA.bin"

It's suggested to use 0x10000 for the "Espressif's official tool" as well, that piece of software was also a bit of a slug not cooperating with me.

Anyway, flashing it gives me an MD5 error.
esptool-v4.2.1-win64>esptool.exe --chip auto --baud 460800 write_flash -z 0x10000 esp32-full.img
esptool.py v4.2.1
Found 1 serial ports
Serial port COM4
Connecting....
Detecting chip type... Unsupported detection protocol, switching and trying again...
Connecting...
Detecting chip type... ESP8266
Chip is ESP8285N08
Features: WiFi, Embedded Flash
Crystal is 26MHz
MAC: 7c:87:ce:a4:00:ae
Uploading stub...
Running stub...
Stub running...
Changing baud rate to 460800
Changed.
Configuring flash size...
Flash will be erased from 0x00010000 to 0x00149fff...
Compressed 1282128 bytes to 760411...
Wrote 1282128 bytes (760411 compressed) at 0x00010000 in 17.9 seconds (effective 573.2 kbit/s)...
File  md5: 6c59c31b4148df44e966cef36b8d06a8
Flash md5: 6712483f773af736c89238e027124d44
MD5 of 0xFF is edf62e3cf87fcc5525719db4c990a136

There also seen to exist a bin in this rep as well, https://github.com/schmurtzm/RFLink32-For-Sonoff-RF-Bridge/blob/master/firmwares/RFLink32-v5.1_Sonoff-RF-Bridge-Direct.bin that i did manage to flash but then nothings happens. 
There is no webgui on it so not sure how to use that, it didn't put up any AP either.

re-flashing it to tasmota with tasmotizer works just fine so the chip ain't broken and/or not working.

**Comment by gurkburk76** [comment]
New info.
I've tried to reflash with some parameters that i found in a bat file:
esptool --chip auto --baud 460800 write_flash -fs 1MB -fm dout 0x0 RFLink32-v5.1_Sonoff-RF-Bridge-Direct.bin

This worked and i could connect to it as it popped open an AP, so that's nice.

Now, https://github.com/schmurtzm/rflink-webui
I've compiled it and it seems to have generated a big JSON file that i'm not sure what to do with, i don't even know if i need it :)

Anyway, i'm not getting any RF info but i haven't yet made to hw mods to this sonos rf bridge, it's the new 2.2 type so i'll have to sodder on some stuff to it...

TBC!
